_posts/2018-03-29-timmie.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  Timmie 
+date:   2018-03-29 10:28:00
+categories: 
+  - projects
+description: a discord bot that manages events
+image: 
+image-sm: 
+author: Joy Sangsrichan
+author-image: https://avatars1.githubusercontent.com/u/31389287?s=460&v=4
+author-bio: First Year Computer Science Student at Rochester Institute of Technology
+author-email: joy.sangsrichan@gmail.com
+author-social:
+  github: https://github.com/jxs2143
+  linkedin: https://www.linkedin.com/in/joy-sangsrichan-390518150/
+---
+Meet Timmie, a discord bot that manages video game events and sends random dog pictures when requested! There is actually a story that Timmie is a robot who believes that he is a knight, hence why he is seen wearing a full Medieval-style armor. 
+
+#### What is Discord?
+Discord is an application that supports voice call, text chat, and screen-sharing designed for gaming communities. One of its features is the ability to add custom bots to the server such as welcome bots, music bots, and administrator bots. With the API wrapper, Discord.py, created by a user on Github named Rapptz, anyone can design and create their own bots. 
+
+#### What can Timmie do?
+Timmie's functions can be separated into four different categories: basic interactions, calculation, event management, and miscellaneous. You can view the full list of functions here on my Github page. 
```

Comment:
"Basic interactions" here isn't really a category of command, since the contents of that section are applicable to all commands. I would remove "basic interactions" as a category and rename the "Basic Interactions" section below to something like "Message handling". Also, is "here" supposed to be a link to your GitHub profile or the repo for Timmie?